data-sample/cmaq/form-ald2-aldx/form-ald2-aldx-update-table.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+source ../../database.cfg
+
+# load cmaq-2010-2011.csv and cmaq_sample.sql to backend container:
+docker cp cmaq-update-2011.csv ${DB_CONTAINER_NAME}:/cmaq-update.csv
```

Comment:
Same as above, comment says 2010-2011, but only 2011 sample data is updated. Just need to keep comment consistent with the command statement.

Reply:
local dev environment only uses 2011 data at the moment.